Microsoft.HealthVault/ItemTypes/ApproximateDate.cs

```diff
@@ -221,7 +233,7 @@ public int Year
             }
         }
 
-        private int _year = DateTime.Now.Year;
+        private int _year = 2000;
```

Comment:
Shouldn't show up in normal use. The parameterless constructor is only there for the XML deserialization and it should get overridden immediately. I wanted to remove a static dependency that the file had.

---
In reply to: [119958581](https://github.com/Microsoft/healthvault-dotnetstandard-sdk/pull/44#discussion_r119958581) [](ancestors = 119958581)